Documentation/ExtensionArchitecture/Extbase/Reference/UriArguments.rst

```diff
@@ -0,0 +1,63 @@
+..  include:: /Includes.rst.txt
+
+..  index:: Extbase; URI arguments; arguments
+
+===================================
+URI arguments and reserved keywords
+===================================
+
+Extbase uses special URI arguments to pass variables
+to Controller arguments and the framework itself.
+
+..  todo: Check for completion of this
+
+Extbase uses a prefixed URI argument scheme that relies
+on plugin configuration.
+
+For example, the example extension `EXT:blog_example` would use:
+
+..  code-block:: plaintext
+
+    https://example.org/blog/?tx_blogexample_bloglist[action]=show&tx_blogexample_bloglist[controller]=Post&tx_blogexample_bloglist[post]=4711&cHash=...
+
+    // Actually, the [] parameters are often URI encoded, so this is emitted:
+    https://example.org/blog/?tx_blogexample_bloglist%5Baction%5D=show&tx_blogexample_bloglist%5Bcontroller%5D=Post&tx_blogexample_bloglist%5Bpost%5D=4711&cHash=...
+
+as the created URI to execute the `showAction` of the Controller `PostController`
+within the plugin `BlogList`.
+
+The following arguments are evaluated:
+
+*   `tx_(extensionName)_(pluginName)[action]`: Controller action to execute
+*   `tx_(extensionName)_(pluginName)[controller]`: Controller containing the action
+*   `tx_(extensionName)_(pluginName)[format]`: Output format (usually `html`, can also be `json` or custom types)
+*   `cHash` - the cHash always gets calculated to validate that the URI is allowed to be called. (see :ref:`chash`)
```

Comment:
```suggestion
 `tx_(extensionName)_(pluginName)[action]`: 
     Controller action to execute
`tx_(extensionName)_(pluginName)[controller]`
    Controller containing the action
`tx_(extensionName)_(pluginName)[format]`
    Output format (usually `html`, can also be `json` or custom types)
`cHash`
    the cHash always gets calculated to validate that the URI is allowed to be called. (see :ref:`chash`)
```
I would suggest to use a definition list for easier readability and better speech support